app/core/src/main/java/com/fsck/k9/job/MailSyncJob.kt

```diff
@@ -16,13 +16,7 @@ class MailSyncJob(
                 ?.let { accountUuid ->
 
                     preferences.getAccount(accountUuid)?.let { account ->
-                        messagingController.checkMail(
```

Comment:
`MailSycJob` finished before the mail check was done. I added the method `MessagingController.checkMailBlocking()` to fix this.

Reply:
👍